ui/pages/import-tokens/import-tokens.js

```diff
@@ -0,0 +1,668 @@
+          standard: tokenStandard,
+        }
+      : null;
+    dispatch(
+      setPendingTokens({ customToken, selectedTokens, tokenAddressList }),
+    );
+    setMode('confirm');
+  };
+
+  const handleCustomAddressChange = async (value) => {
+    const address = value.trim();
+
+    setCustomAddress(address);
+    setCustomAddressError(null);
+    setNftAddressError(null);
+    setSymbolAutoFilled(false);
+    setDecimalAutoFilled(false);
+    setMainnetTokenWarning(null);
+
+    const addressIsValid = isValidHexAddress(address, {
+      allowNonPrefixed: false,
+    });
+    const standardAddress = addHexPrefix(address).toLowerCase();
+
+    const isMainnetToken = Object.keys(STATIC_MAINNET_TOKEN_LIST).some(
+      (key) => key.toLowerCase() === address.toLowerCase(),
+    );
+
+    let standard;
+    if (addressIsValid) {
+      try {
+        ({ standard } = await getTokenStandardAndDetails(
+          standardAddress,
+          selectedAddress,
+          null,
+        ));
+      } catch (error) {
+        // ignore
+      }
+    }
+
+    const addressIsEmpty = address.length === 0 || address === EMPTY_ADDRESS;
+
+    switch (true) {
+      case !addressIsValid && !addressIsEmpty:
+        setCustomAddressError(t('invalidAddress'));
+        setCustomSymbol('');
+        setCustomDecimals(0);
+        setCustomSymbolError(null);
+        setCustomDecimalsError(null);
+        break;
+
+      case standard === TokenStandard.ERC1155 ||
+        standard === TokenStandard.ERC721:
+        setNftAddressError(
+          t('nftAddressError', [
+            <ButtonLink
+              className="import-tokens-modal__nft-address-error-link"
+              onClick={() => {
+                dispatch(showImportNftsModal({ tokenAddress: address }));
+                // nClose();
+                history.push(mostRecentOverviewPage);
+              }}
+              color={TextColor.primaryDefault}
+              key="nftAddressError"
+            >
+              {t('importNFTPage')}
+            </ButtonLink>,
+          ]),
+        );
+        break;
+
+      case isMainnetToken && !isMainnet:
+        setMainnetTokenWarning(t('mainnetToken'));
+        setCustomSymbol('');
+        setCustomDecimals(0);
+        setCustomSymbolError(null);
+        setCustomDecimalsError(null);
+        break;
+
+      case Boolean(identities[standardAddress]):
+        setCustomAddressError(t('personalAddressDetected'));
+        break;
+
+      case checkExistingAddresses(address, tokens):
+        setCustomAddressError(t('tokenAlreadyAdded'));
+        break;
+
+      default:
+        if (!addressIsEmpty) {
+          attemptToAutoFillTokenParams(address);
+          if (standard) {
+            setTokenStandard(standard);
+          }
+        }
+    }
+  };
+
+  // Determines whether to show the Search/Import or Confirm action
+  const isConfirming = mode === 'confirm';
+
+  return (
+    <div className="page-container">
```

Comment:
Hello @sahar-fehri !  Instead of using `<div>`, we should be using the Design System `<Box>` component and its props to put together these layouts.  Let me know if you have any questions with Box.  Its props have loads of use, like `display`, etc.

Reply:
Hey @darkwing ! Thanks for your comment; i have replaced the div with Box on this page and also on token-list.component.js. 
I wanted to use the props instead of the classNames but that didn't make the CSS look like i want; so i kept the classNames